backend/src/SerialHandler.py

```diff
@@ -69,7 +70,10 @@ def _get_serial_message(self):
         """
         Get the serial message from the serial port.
         """
-        return self.serial_port.read_until(expected = b'\x00', size = None)
+        try:
+            return self.serial_port.read_until(expected = b'\x00', size = None)
+        except Exception:
+            return None
```

Comment:
Should we be handling this? Also what would cause an exception in this case?

Reply:
This handling does make sense for how cobs are encoded since a message will always end with a 0

for the exception, the UART has a builtin timeout which isn't really a concern since we will just not receive a message implying that the serial port is halted on the other, or just not sending actually valuable data (garbage).